src/config/example.config.js

```diff
@@ -0,0 +1,5 @@
+// Put your API key here!
```

Comment:
I think we could go ahead on delete this file.. I think it's deleted on main, so not sure if this merge would re-add it?